src/main/java/com/arpnetworking/metrics/common/sources/CollectdHttpSourceV1.java

```diff
@@ -45,6 +45,7 @@ private CollectdHttpSourceV1(final Builder builder) {
         public Builder() {
             super(CollectdHttpSourceV1::new);
             setParser(new CollectdJsonToRecordParser());
+            setActorName("collectdv1");
```

Comment:
Should this be a public static final somewhere since it's referred to at least in two places?